Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 54;
+	objectVersion = 51;
```

Comment:
I don't think we should be making changes to `objectVersion`.

cc @emw-apple

Reply:
`54` is the archiver version for Big Sur-aligned Xcode, which is the oldest Apple platform we support, so it should be safe to use here.

Reply:
I'm gonna remove it anyway, it should not be in this PR.